jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -525,16 +531,23 @@ static int fastIdentityHashCode(Object anObject) {
 	static int valueHashCode(Object anObject) {
 		int hashcode;
 
+		if (null == valueObjectHashCode) {
+			try {
+				valueObjectMethods = Class.forName("java.lang.runtime.ValueObjectMethods");
```

Comment:
If `valueObjectMethods` is used only once here, there is no need to cache it into a static variable.

Reply:
updated in 144af13